backend/src/test/java/ch/puzzle/okr/service/ObjectiveServiceTest.java

```diff
@@ -53,4 +68,24 @@ void shouldReturnEmptyListWhenNoObjective() {
 
         assertEquals(0 ,objectives.size());
     }
+
+    @Test
+    void shouldGetAllKeyresultsByObjective() {
+        Mockito.when(objectiveRepository.findById(1L)).thenReturn(Optional.ofNullable(objective));
```

Comment:
Verwende statt `Optional.ofNullable(objective)` folgendes `Optional.of(objective)`. Ich werde am Montag erklären, weshalb.